### PR TITLE
Add Mario & Luigi Superstar Saga + Bowser's Minions

### PR DESCRIPTION
**Pull request by Sedhaild**
Adds Mario & Luigi Superstar Saga + Bowser's Minions.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/mario-luigi-superstar-saga-bowsers-minios/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Mario & Luigi Superstar Saga + Bowser's Minios"
```
Comment (Schplee) [inline games/mario-luigi-superstar-saga-bowsers-minios/game.dat]: `Minions` is spelled wrong.

File: games/mario-luigi-superstar-saga-bowsers-minios/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+description = "Platform: Nintendo 3DS\nRelease Date: Oct 06, 2017\nNo. of Players: 1 player\nCategory: Action, Adventure, Role-Playing\nPublisher: Nintendo\nDeveloper: ALPHADREAM"
```
Comment (Schplee) [inline games/mario-luigi-superstar-saga-bowsers-minios/game.dat]: Incorrect description. Should be the first couple sentences from the wiki article, not a copy and paste from the sidebar.

**Comment by Sedhaild** [comment]
Corrections done!

File: games/mario-luigi-superstar-saga-bowsers-minios/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+region = "eur"
+release_date = "2017-10-06"
+
+[[testcases]]
+title = "00040000001B9000"
+compatibility = "0"
+date = "2017-10-30"
+version = "HEAD-1f6da9f"
+cpu = "Intel Core i5-6400"
+gpu = "GeForce GTX 750TI"
```
Comment (Leo626) [inline games/mario-luigi-superstar-saga-bowsers-minios/game.dat]: `Nvidia`

File: games/mario-luigi-superstar-saga-bowsers-minios/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
+title = "Mario & Luigi Superstar Saga + Bowser's Minions"
+description = "Mario & Luigi: Superstar Saga + Bowser's Minions is a game for the Nintendo 3DS. It is an enhanced remake of the 2003 Game Boy Advance game, Mario & Luigi: Superstar Saga. The game has an added story called Minion Quest: The Search for Bowser."
```
Comment (Leo626) [inline games/mario-luigi-superstar-saga-bowsers-minios/game.dat]: Small nit, can you mention the developer at the end of the first sentence:
```
Mario & Luigi: Superstar Saga + Bowser's Minions is a game for the Nintendo 3DS, developed by AlphaDream.
```

File: games/mario-luigi-superstar-saga-bowsers-minios/game.dat (hunk 3 above)
Comment (Sedhaild) [reply games/mario-luigi-superstar-saga-bowsers-minios/game.dat]: added.

File: games/mario-luigi-superstar-saga-bowsers-minios/game.dat (hunk 4 above)
Comment (Sedhaild) [reply games/mario-luigi-superstar-saga-bowsers-minios/game.dat]: done!

**Comment by Leo626** [comment]
In the directory `mario-luigi-superstar-saga-bowsers-minios` , `minions` is spelled wrong.

**Comment by Sedhaild** [comment]
Directory's name corrected... I think...

**Pull request merged**